db/models.py

```diff
@@ -22,6 +29,11 @@ class Movie(models.Model):
     actors = models.ManyToManyField(to=Actor, related_name="movies")
     genres = models.ManyToManyField(to=Genre, related_name="movies")
```

Comment:
The `related_name` attribute in the `ManyToManyField` for `genres` is set to 'movies'. Similar to the previous comment, ensure that this related name is unique across your models to prevent conflicts.